deploy-asm.sh

```diff
@@ -50,7 +50,6 @@ sleep 5s
   --cluster_location $LOCATION \
   --fleet_id $PROJECT \
   --output_dir asmoutput \
-  --option cloud-tracing \
```

Comment:
@danistrebel - I dont see the error on my end. Since we are not doing tracing (yet), we can remove this

Reply:
the --option flag refers to an override file. That file isn't supplied so the asmcli command crashes with an exit 22.

Reply:
as per the docs, it will go to a default directory and pick the cloud-tracing.yaml file
It worked for me and @micovery